src/DynamoPackages/Package.cs

```diff
@@ -164,6 +164,7 @@ public static Package FromJson(string headerPath, ILogger logger)
                 if (body.name == null || body.version == null)
                     throw new Exception("The header is missing a name or version field.");
 
+                // TODO: Add serialization part of hosts specified in package.json
```

Comment:
did you verify if this now gets serialized?

Reply:
The property is serialized but it is empty. e.g.
```Json
{
    "license": "",
    "file_hash": null,
    "name": "tesingDepIssues",
    "version": "0.0.1",
    "description": "tesingDepIssues",
    "group": "",
    "keywords": null,
    "dependencies": [],
    "contents": "testingDepIssues - upload civil",
    "engine_version": "2.1.1.896",
    "engine": "dynamo",
    "engine_metadata": "",
    "site_url": "",
    "repository_url": "",
    "contains_binaries": false,
    "node_libraries": []
}
```

Reply:
I don't see the `host` property there - `dependencies` is not the same property - it marks dependencies on other packages.

Reply:
Sorry I was confused by myself, so my TODO comment is valid then :)